cmd/kubelet/app/options/container_runtime.go

```diff
@@ -50,6 +66,7 @@ func NewContainerRuntimeOptions() *config.ContainerRuntimeOptions {
 		DockerEndpoint:             dockerEndpoint,
 		DockershimRootDirectory:    "/var/lib/dockershim",
 		PodSandboxImage:            defaultPodSandboxImage,
+		PodSandboxSeccomp:          defaultPodSandboxSeccomp,
```

Comment:
@tallclair here is where the pod sandbox seccomp is being set when dockershim is not being used.